src/components/errors/layout.scss

```diff
@@ -6,10 +6,10 @@
 .ed-errors {
     padding: $mmf-unit 2 * $mmf-unit;
     font-size: $mmf-text-size--meta;
-    color: $mmf-color--error;
+    color: map-get($color-danger, 2);
 
     .is-warning {
-        color: $mmf-color--warning;
+        color: map-get($color-danger, 3);
```

Comment:
Keine warning-color mehr?

Reply:
Doch 😄  ist die gleiche Color. Wir haben uns darauf geeinigt in mmf auf die gleiche Weise colors zu setzten wie in der lux-config. Also mit map-get und dann mit IDs. Die _color--warning_ ist also die gleiche wie die _color-danger_ mit _id 3_.